extensions/amp-story/0.1/amp-story-desktop.css

```diff
@@ -344,6 +378,11 @@ amp-story-page.i-amphtml-story-page-shown {
  bottom: 0;
 }
 
+[desktop] .i-amphtml-story-page-progress-bar {
+  margin: 0 2px;
+  overflow: hidden;
```

Comment:
We should move this to `amp-story.css`, since it applies to desktop and mobile.

Reply:
Done